apps/fabric-website/src/pages/Controls/ListCells/ListCellsPage.doc.ts

```diff
@@ -0,0 +1,13 @@
+import { TFabricPlatformPageProps } from '../../../interfaces/Platforms';
```

Comment:
Looks like I messed up the naming of this folder, and that's why the build failed--the folder should be `ListCellsPage`, not `ListCells`.

Reply:
Thanks! Fixing locally.